Source/player.cpp

```diff
@@ -49,53 +49,17 @@ int plrxoff2[9] = { 0, 1, 0, 1, 2, 0, 1, 2, 2 };
 /** Specifies the Y-coordinate delta from a player, used for instance when casting resurrect. */
 int plryoff2[9] = { 0, 0, 1, 1, 0, 2, 2, 1, 2 };
 
-/** Maps from player_class to starting stat in strength. */
-int StrengthTbl[enum_size<HeroClass>::value] = {
-	30,
-	20,
-	15,
-	25,
-	20,
-	40,
-};
-/** Maps from player_class to starting stat in magic. */
-int MagicTbl[enum_size<HeroClass>::value] = {
+/** Maps from player_class to starting stat. */

```

Comment:
0 info about class and stats, everything accessed just by raw index, burn it with fire 🤮 
add comments with stat / class name + maybe use some enum for accessing the array?  
0 = STAT_STR or something

Reply:
how's that?